dags/job_label.py

```diff
@@ -0,0 +1,45 @@
+import csv
```

Comment:
Hound isn't doing its job here, but there seem to be a number of unnecessary imports.